pkg/settlement/swap/chequebook/chequebook.go

```diff
@@ -135,3 +144,44 @@ func (s *service) WaitForDeposit(ctx context.Context, txHash common.Hash) error
 	}
 	return nil
 }
+
+// IssueCheque issues a new cheque
+func (s *service) IssueCheque(beneficiary common.Address, amount *big.Int) (*SignedCheque, error) {
```

Comment:
i wonder if we need the endpoint to be `IssueCheque`, since it is in the chequebook already, so you would usually call some `chequebook.IssueCheque()`, maybe `chequebook.Issue()` is enough in that sense.

Reply:
makes sense. changed.